src/Event/Browser.Event.fs

```diff
@@ -33,10 +33,20 @@ type [<AllowNullLiteral>] EventType =
     abstract BUBBLING_PHASE: float with get, set
     abstract CAPTURING_PHASE: float with get, set
 
+type [<AllowNullLiteral>] AddEventListenerOptions =
```

Comment:
is this the favored form for options? I thought about an anon record but I wasn't sure that would be ergonomic.

Reply:
We indeed use interface for building option types because with the combination of  `jsEmpty` or `jsOptions` user can just provide the needed properties and it will be transformed into a POJO.

Using a record or anon here would require the user to set all the properties which is not what we want in general.